hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHome.java

```diff
@@ -55,8 +54,16 @@
     public PhoneHome(Node node) {
         hazelcastNode = node;
         logger = hazelcastNode.getLogger(com.hazelcast.internal.util.phonehome.PhoneHome.class);
+        basePhoneHomeUrl = "http://phonehome.hazelcast.com/ping";
```

Comment:
Please change this constructor so that it just calls the other 2-arg constructor. Generally it is good to keep all the field assignments in a single constructor (as long as possible), so that if any argument validation is necessary, it is enough to do that at one place only.

Also please extract the `"http://phonehome.hazelcast.com/ping"` value into a constant called `DEFAULT_BASE_PHONE_HOME_URL`.